website/views/index.jade

```diff
@@ -13,34 +13,43 @@ html(ng-app="habitrpg", ng-controller="RootCtrl", ng-class='{"applying-action":a
     meta(name='apple-mobile-web-app-capable', content='yes')
     meta(name='mobile-web-app-capable', content='yes')
 
-    include ./shared/new-relic
-    //FIXME for some reason this won't load when in footerCtrl.js#deferredScripts()
-    script(type="text/javascript", src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-5016f6cc44ad68a4", async="async")
+    != env.getManifestFiles("app", "css")
```

Comment:
When we do the client refactor, we should figure out what we want our code style to be. https://github.com/pugjs/pug-lint